### ISSUE TITLE
Customization for Objective functions

### ISSUE THREAD

**Issue by windweller** [fchollet/keras#1019]
Hi, I'm trying to replicate Ian Goodfellow's ICLR 2015 paper on Adversarial Learning. His paper demonstrated a custom objective function that was built on top of the normal LSTM cost function. Can I supplement my own custom objective function in Keras??

**Comment by fchollet** [comment]
Sure, you can write your own loss functions. Check out keras/objectives.py
for some examples.
